source/codegen/metadata/niscope/enums.py

```diff
@@ -695,217 +579,130 @@
             }
         ]
     },
-    'FIRFilterWindow': {
```

Comment:
Missing

Reply:
same as above

Reply:
This has been added back since its referenced by an attribute now